webapp/src/components/shareBoard/channelPermissionsRow.tsx

```diff
@@ -44,20 +43,7 @@ const ChannelPermissionsRow = (props: Props): JSX.Element => {
     }
 
     useEffect(() => {
-        if (!Utils.isFocalboardPlugin() || !board.channelId) {
-            setLinkedChannel(null)
-            return
-        }
-        const unknownChannel = {
-            id: board.channelId,
-            type: 'P',
-            name: 'unknown',
-            display_name: intl.formatMessage({
-                id: 'shareBoard.unknown-channel-display-name',
-                defaultMessage: 'Unknown channel',
-            }),
-        } as Channel
-        octoClient.getChannel(board.teamId, board.channelId).then((c) => setLinkedChannel(c || unknownChannel))
+        setLinkedChannel(null)
     }, [board.channelId])
```

Comment:
`board.channelId` can be removed from dependency list now as we're no longer using it.

Reply:
I think we don't required the completed channelPermissionsRow component, as channelPermissions are not been used in personal or standalone focalboard application.